File: A_Vanya_and_Fence.py
n, h = map(int,input().split())
row = 0
while(n!=0):
    fh = list(int,input().split())
    if(fh <= h):
        row = row + 1
    else:
        row = row + 2
    n = n - 1
print(row)

File: A_Vanya_and_Fence.c
#include<stdio.h>
int main(){
    int n, k, i;
    int frnd = n;
    scanf("%d %d", &n, &k);
    for (i = 0; i < n;i++){
        scanf("%d",&row);
        frnd = frnd + (row > k);
    }
    printf("%d", frnd);
}